test/functional/sqlsrv/TC34_PrepAndExec.phpt

```diff
@@ -9,114 +9,101 @@ PHPT_EXEC=true
         }
         $values[$i] = $fieldVal;
     }
     sqlsrv_free_stmt($stmt1);
-    Trace(" $numFields1 fields retrieved.\n"); 
+    trace(" $numFields1 fields retrieved.\n");
 
     // Prepare once and execute several times
-    Trace("Prepare a SELECT query on $tableName ...");
+    trace("Prepare a SELECT query on $tableName ...");
     $stmt2 = PrepareQuery($conn1, "SELECT * FROM [$tableName]");
     $numFields2 = sqlsrv_num_fields($stmt2);
-    Trace(" $numFields2 fields expected.\n"); 
-    if ($numFields2 != $numFields1)
-    {
-        SetUTF8Data(false);
+    trace(" $numFields2 fields expected.\n");
+    if ($numFields2 != $numFields1) {
+        setUTF8Data(false);
         die("Incorrect number of fields: $numFields2");
     }
 
-    for ($j = 0; $j < $noPasses; $j++)
-    {
-        Trace("Executing the prepared query ...");
+    for ($j = 0; $j < $noPasses; $j++) {
+        trace("Executing the prepared query ...");
         sqlsrv_execute($stmt2);
         sqlsrv_fetch($stmt2);
-        for ($i = 0; $i < $numFields2; $i++)
-        {
-            if (UseUTF8Data()){
+        for ($i = 0; $i < $numFields2; $i++) {
+            if (useUTF8Data()) {
                 $fieldVal = sqlsrv_get_field($stmt2, $i, SQLSRV_PHPTYPE_STRING('UTF-8'));
-            }
-            else{
+            } else {
                 $fieldVal = sqlsrv_get_field($stmt2, $i, SQLSRV_PHPTYPE_STRING(SQLSRV_ENC_CHAR));
             }
-            if ($fieldVal === false)
-            {
-                FatalError("Failed to retrieve field $i");
+            if ($fieldVal === false) {
+                fatalError("Failed to retrieve field $i");
             }
-            if ($values[$i] != $fieldVal)
-            {
-                SetUTF8Data(false);
+            if ($values[$i] != $fieldVal) {
+                setUTF8Data(false);
                 die("Incorrect value for field $i at iteration $j");
             }
         }
-        Trace(" $numFields2 fields verified.\n"); 
+        trace(" $numFields2 fields verified.\n");
     }
     sqlsrv_free_stmt($stmt2);
 
-    DropTable($conn1, $tableName);  
-    
+    dropTable($conn1, $tableName);
+
     sqlsrv_close($conn1);
 
-    EndTest($testName); 
+    endTest($testName);
 }
 
 //--------------------------------------------------------------------
-// Repro
+// repro
 //
 //--------------------------------------------------------------------
-function Repro()
+function repro()
 {
-    if (! IsWindows())
-    {
-        SetUTF8Data(true);
+    if (! isWindows()) {
+        setUTF8Data(true);
     }
-    try
-    {
+    try {
         PrepareAndExecute(5);
-    }
-    catch (Exception $e)
-    {
+    } catch (Exception $e) {
         echo $e->getMessage();
     }
-    SetUTF8Data(false);
+    setUTF8Data(false);
 }
 
-Repro();
+repro();
 
 ?>
 --EXPECT--
-Test "Statement - Prepare and Execute" completed successfully.
\ No newline at end of file
+Test "Statement - Prepare and Execute" completed successfully.
```

Comment:
this comment is not to request you to change anything in this PR. But when you're refactoring the tests, can you make sure trace(), startTests and endTest are no longer used. Also, any test that only have one repro() at the end, change them so the test goes to the repro directly.

Reply:
I can remove repro() but some tests do use trace() sensibly, so I opted to keep them. About startTests() and endTests() I don't mind keeping them around either unless you really want them removed. :)